CHANGELOG.md

```diff
@@ -1,3 +1,51 @@
+# 1.0.10-rc1 (2020-06-23)
+## Edge Agent
+### Features
```

Comment:
I am not seeing module boot order and NVida support through moby..

Reply:
Yes, +1 on module boot order.

Reply:
added.